cmd/crowdsec-cli/install.go

```diff
@@ -63,6 +64,24 @@ you should [update cscli](./cscli_update.md).
 			if !config.configured {
 				return fmt.Errorf("you must configure cli before interacting with hub")
 			}
+
+			/*
+				if no branch has been specified in flags for the hub, then use the one corresponding to crowdsec version
+			*/
+			if cwhub.HubBranch == "" {
+				latest, err := cwversion.Latest()
+				if err != nil {
+					log.Fatalf("unable to get last crowdsec version: %s", err)
+				}
+
+				if cwversion.Version == latest {
+					cwhub.HubBranch = "master"
+				} else {
+					log.Warnf("Crowdsec is not the latest version. Current version is '%s' and latest version is '%s'. Please update it!", cwversion.Version, latest)
+					cwhub.HubBranch = cwversion.Version
```

Comment:
maybe we want to specify to which branch we're going to stick if it's not master ?

Reply:
I think this is what the ligne above does no ?

Reply:
mmh my bad, it doesn't precise the branch, but says what is the current crowdsec version and what is the latest. Is it enough ?